docs/UserGuide.md

```diff
@@ -196,30 +196,37 @@ Examples:
 The find command has different behaviours depending on the current list:
 
 1. In `STUDENTS` list:
-    - Finds students whose names contain any of the given keywords.
+    - Finds students whose names are made up of the given search keyword.
 2. In `SCHEDULE` list:
-    - TBC
+    - Finds lessons whose names are made up of the given search keyword.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `find KEYWORD`
 
 1. In `STUDENTS` list:
    * The search is case-insensitive. e.g `hans` will match `Hans`
-   * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
    * Only the name is searched.
-   * Only full words will be matched e.g. `Han` will not match `Hans`
-   * Persons matching at least one keyword will be returned (i.e. `OR` search).
-     e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+   * Persons matching part of the keyword will be returned (i.e. `OR` search).
+     e.g. `Hans` will return `Hanso Gruber`, `Hansel`
+   * Keyword specified has to be the first part of the name
+     e.g. `Hans` will return `Hanso Gruber` but not `Lee Hans`
 2. In `SCHEDULE` list:
-    * TBC
+    * The search is case-insensitive. e.g `lesson` will match `Lesson`
+    * Only the name is searched.
+    * Lessons matching part of the keyword will be returned (i.e. `OR` search).
+      e.g. `Lesson Chem` will return `Lesson Chemistry`, `Lesson Chem at Bishan`
+    * Keyword specified has to be the first part of the name
+        e.g. `Blk 12` will return `Blk 123 Bedok Chemistry` but not `Bishan Blk 12`
```

Comment:
both actually returns
because it simply searches if the lesson name contains the search string

Reply:
okies made the necessary changes in the most recent commit